src/MultiDocumenter.jl

```diff
@@ -57,25 +72,16 @@ end
 function walk_outputs(f, root, docs::Vector{MultiDocRef}, dirs::Vector{String})
     for ref in docs
         p = joinpath(root, ref.path)
-        for dir in dirs
-            dirpath = joinpath(p, dir)
-            isdir(dirpath) || continue
-            for (r, _, files) in walkdir(dirpath)
-                for file in files
-                    file == "index.html" || continue
```

Comment:
@pfitzseb Do you know why we're filtering for `index.html` and not just for any HTML files? This would exclude builds that don't have pretty URLs enabled etc.

Reply:
Yep, let's switch to a `DocumenterTools.isdochtml` check in a follow-up.